docs/polymorphism.md

```diff
@@ -803,25 +810,49 @@ abstract class Project {
     abstract val name: String
 }
 
-@Serializable
-data class BasicProject(override val name: String, val type: String): Project()
-
 @Serializable
 @SerialName("OwnedProject")
 data class OwnedProject(override val name: String, val owner: String) : Project()
+
```

Comment:
I'd suggest to move it to a separate section named "Maintaining custom attributes" in the chapter on "JSON elements". That is, give a simpler example here, explain its disadvantage, and give a link to another section. Otherwise, people reading the guide top-to-bottom will not be able to understand what's going on here.